apps/test-bundles/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "node ../../scripts/build.js",
+    "build": "node ../../scripts/build.js && node ../../scripts/bundle-size-collect.js",
```

Comment:
This doesn't seem like something that should run for every build.

Reply:
This probably should only run in... production builds or CI workflows?

Reply:
I have removed the invocation from `apps/test-bundles/package.json`. This script shall be invoked as a step in the build definition on Azure Dev Ops as `node scripts/bundle-size-collect.js`. Since the script would be run from the root of the repository on Azure Dev Ops, I have changed the `distRoot` path in the script to be `'apps/test-bundles/dist'`